helper-cli/src/main/kotlin/commands/GenerateScopeExcludesCommand.kt

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright (C) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.helper.commands
+
+import com.beust.jcommander.JCommander
+import com.beust.jcommander.Parameter
+import com.beust.jcommander.Parameters
+
+import com.here.ort.CommandWithHelp
+import com.here.ort.helper.common.greedySetCover
+import com.here.ort.helper.common.replaceScopeExcludes
+import com.here.ort.helper.common.sortScopeExcludes
+import com.here.ort.helper.common.writeAsYaml
+import com.here.ort.model.OrtResult
+import com.here.ort.model.config.RepositoryConfiguration
+import com.here.ort.model.config.ScopeExclude
+import com.here.ort.model.config.ScopeExcludeReason
+import com.here.ort.model.readValue
+import com.here.ort.utils.PARAMETER_ORDER_MANDATORY
+import com.here.ort.utils.PARAMETER_ORDER_OPTIONAL
+
+import java.io.File
+
+@Parameters(
+    commandNames = ["generate-scope-excludes"],
+    commandDescription = "Generate scope excludes."
```

Comment:
"... based on common default for the package managers."

Reply:
done.